cli/compose/types/types.go

```diff
@@ -80,6 +80,7 @@ type ServiceConfig struct {
 	CgroupParent    string   `mapstructure:"cgroup_parent"`
 	Command         ShellCommand
 	ContainerName   string   `mapstructure:"container_name"`
+	CredentialSpec  string     `mapstructure:"credential_spec"`
```

Comment:
@aluzzardi we could embed, but you're never gonna use credential_spec with the selinux stuff anyway, so I don't see why.